cluster/src/dunit/scala/io/snappydata/cluster/SplitSnappyClusterDUnitTest.scala

```diff
@@ -140,7 +140,7 @@ object SplitSnappyClusterDUnitTest extends SplitClusterDUnitTestObject {
     val snc = SnappyContext(sc)
     // remove below once SNAP-653 is fixed
     val numPartitions = props.getOrElse("buckets", "113").toInt
-    StoreUtils.removeCachedObjects(snc, "EMBEDDEDMODETABLE1", numPartitions,
+    StoreUtils.removeCachedObjects(snc, "APP.EMBEDDEDMODETABLE1", numPartitions,
```

Comment:
it is a product  utility method  which takes the qualifiedTableName and now the table name is always going to be fullyQualified even if user provides a table name without schema.